src/main/java/net/minestom/server/extras/selfmodification/MinestomExtensionClassLoader.java

```diff
@@ -105,8 +105,14 @@ protected void finalize() throws Throwable {
      * @see MinestomRootClassLoader#loadBytes(String, boolean) for more information
      */
     protected boolean isMainExtensionClass(String name) {
-        if(mainClassName.equals(name))
+
+        if (mainClassName.equals(name))
             return true;
-        return children.stream().anyMatch(c -> c.isMainExtensionClass(name));
+
+        for (MinestomExtensionClassLoader child : children) {
+            if (child.isMainExtensionClass(name)) return true;
+        }
+
+        return false;
```

Comment:
While I don't mind the change, I'm curious as to why you changed this to something that has the same semantic?

Reply:
I'm personally not a big fan of streams, though I'm not sure where streams stand in Minestom -- like we mainly use for loops but theres streams (mainly for testing / debugging) so I wanted to follow that with this.

Reply:
I believe that the stream in this case is cleaner